ui/sortable.js

```diff
@@ -830,6 +831,16 @@ return $.widget("ui.sortable", $.ui.mouse, {
 
 	},
 
+	_createTrPlaceholder: function(that, sourceTr, targetTr) {
+		that = that || this;
```

Comment:
The first parameter is unnecessary. You can change this line to `var that = this;`.